static/docs/commands-reference/add.md

```diff
@@ -96,25 +96,25 @@ This way you bring data provenance and make your project reproducible.
 Take a file under DVC control:
 
 ```dvc
-    $ dvc add data.xml
+$ dvc add data.xml
 
-    Adding 'data.xml' to '.gitignore'.
-    Saving 'data.xml' to cache '.dvc/cache'.
-    Saving information to 'data.xml.dvc'.
+Adding 'data.xml' to '.gitignore'.
+Saving 'data.xml' to cache '.dvc/cache'.
+Saving information to 'data.xml.dvc'.
 
-    To track the changes with git run:
+To track the changes with git run:
 
-    	git add .gitignore data.xml.dvc
+	git add .gitignore data.xml.dvc
```

Comment:
is it a tab symbol here? don't see any changes

Reply:
Except DVC blocks, any other changes to indentation needed?
Let me know. :+1: